__tests__/app.test.js

```diff
@@ -73,3 +73,51 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("get reviews by id", () => {
+  test("for an api call for a review ID that is given as a non integer, should return 400 and an invalid ID message", () => {
+    return request(app)
+      .get("/api/reviews/hello")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid query review ID must be int");
+      });
+  });
+
+  test("for an api call for a review_id that is an int but has no response as there are no reviews associated with it ", () => {
+    return request(app)
+      .get("/api/reviews/78")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("review ID not found");
+      });
+  });
+
+  test("should return associated reviews when given a valid ID", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(Array.isArray(review)).toBe(true);
+        expect(review.length).toBe(1);
+        expect(review[0] && typeof review[0] === "object").toBe(true);
+        expect(review[0].review_id).toBe(2)
```

Comment:
This line is unnecessary, you can just make your review_id check on line 108 to make sure the review ID is 2.